ESLint/VSESLINTpackage.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Ezt viszont már (az .eslintrc-vel ellentétben) átmásolhatnánk a `$(SolutionDir)`-be. Lásd a .NET Analyzers Build.props-át is az .editorconfiggal. Mert ez mindenhol ugyanaz kell, hogy legyen, és együtt kell frissülnie a GE package.json-jával, ugye? Mert ha igen, akkor a `devDependencies`-nek is meg kellene egyeznie, az összes verziónak.

A docot, .gitignore-t ennek megfelelően kell szintén frissíteni.

Reply:
ebbe nem kell minden ami a GE package.json-jában van

Reply:
Azt értem :). Viszont ezt a fájlt (nem a Ge package.json-t) lehetne másolni, és a `devDependencies` verzióinak nem kellene eltérni.

Reply:
'^' miatt igazából mindegy is volt a verzió eltérés, docsba kerüljön erről valami, hogy ez is legyen frissítve ha a GE packageben is van nagyverziószám frissítés az eslinteknél?

Reply:
Gondolom nem jobb attól viszont, hogy eltér, szal akkor legyen ugyanaz. És igen, legyen légyszi.

Reply:
Azért nem a `$(SolutionDir)`-be másolja, mert abba a mappába kell amiben a `node_modules` van, és azt meg a `Gulp.Extensions.csproj` bemásolja `../../node_modules`-ba, úgyhogy ez is `../../`-ba megy